tests/test_browser_model.py

```diff
@@ -36,7 +36,15 @@ def test_close_window(foxpuppet):
 def test_switch_to(foxpuppet, selenium):
     """Test Switch to function"""
     foxpuppet.browser.open_window()
-
     # Switch to originally window opened by pytest
     foxpuppet.browser.switch_to()
     assert foxpuppet.browser.handle == selenium.current_window_handle
+
+
+def test_tracking_protection_sheild(foxpuppet, selenium):
+    """Tests if the tracking protection icon displays"""
+    browser = foxpuppet.browser.open_window(private=True)
+    browser.switch_to()
+    assert not browser.is_tracking_shield_displayed
```

Comment:
@rbillings what are your requirements for the tracking shield support? What has to be covered?